client/app/scripts/liveblog-edit/module.js

```diff
@@ -11,581 +11,14 @@
 define([
     'angular',
     'lodash',
+    './blog-edit.controller',
```

Comment:
I know I should have said that earlier, but could it be possible to have a `controllers/` folder and put all the controllers in there?